crates/biome_js_analyze/tests/specs/nursery/useAriaPropsSupportedByRole/valid.jsx

```diff
@@ -23,7 +23,6 @@
 <a href="#" aria-live />
 <a href="#" aria-owns />
 <a href="#" aria-relevant />
-<a aria-checked />
```

Comment:
This is one of the case where our implementation diverges from the ESLint one.
The ESLint rule [allows any aria attribute when it is not able to determine a role for an element](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/rules/role-supports-aria-props.js#L55).

We diverge in two ways:
- Our code assigns the `generic` role to `a` elements without `href`.
  Conversely, `aria-query` doesn't assign any implicit role.
  I don't know which approach is the best.
  This is something we should investigate
- Our rule doesn't give up if a role cannot be determined. Instead, we report any non-global aria attribute as invalid.